app/javascript/packs/application.js

```diff
@@ -1,9 +1,11 @@
 import main from '../mastodon/main';
 
 if (!window.Intl || !Object.assign || !Number.isNaN ||
-    !window.Symbol || !Array.prototype.includes) {
+  !window.Symbol || !Array.prototype.includes) {
   // load polyfills dynamically
-  import('../mastodon/polyfills').then(main);
+  require('intl');
+  require('intl/locale-data/jsonp/en.js');
+  import('../mastodon/polyfills').then(main());
```

Comment:
This is not doing what you might think it's doing. This will execute the `main()` function immediately rather than waiting for the promise to resolve (I wrote [a blog post about this](https://pouchdb.com/2015/05/18/we-have-a-problem-with-promises.html) in case it helps :smile:).